Cloud Application Manager/Core Concepts/providers.md

```diff
@@ -31,6 +33,40 @@ An active Cloud Application Manager account.
 
 A provider is a public or private cloud account you register in Cloud Application Manager. Cloud Application Manager interfaces with the provider account’s API to provision and orchestrate deployments automatically. Before you can deploy workloads from Cloud Application Manager, you need to register a provider first.
 
+### Synchronization process
+
+Cloud Application Manager requires to gather specific information of your cloud providers to be able to provision and orchestrate deployments automatically. The basic synchronization of a provider created from an existing account is a read only process which do not change anything in your cloud account.
+
+After you register a provider, Cloud Application Manager uses API calls to retrieve metadata which describes available infrastructure options, quotas, your custom configuration and images or templates to store it with the purpose of provide such customized options to you when you are creating or editing a deployment policy box.
+
+This metadata includes required infrastructure data as follows:
+* Available regions or datacenters
+* Available networks and subnetworks
+* Allowed connectivity options with public or private ip and firewall rules
+* Available autoscalability configuration (i.e. resource limits or alarms)
+* Allowed type/size of virtual machines
+* Available images or templates to be used at provision time. These are listed in configuration of your provider.
+* Available public keys to secure provisioned virtual machines
+* Available load balancers and target groups
+* Unregistered virtual machines and other cloud resources which are already provisioned in your cloud account in order to allow you to register those virtual machines or resources if you want to do so. These are listed in Unregistered tab of your provider. Discovery of these unregistered resources can be done atomatically if you selected so in your provider Services tab: 'Automatic Discovery of Resources'
```

Comment:
I would rather specify that the "registrable" resources are VMs and scaling groups, not any resource. Those are the ones displayed under "Unregistered resources" tab. But the rest of the native resources are listed in the Instances page (for AWS and Azure for now)
Please, also add a link to 'Automatic Discovery of Resources' that is explained in AWS or Azure providers

Reply:
done